azure-pipelines.yml

```diff
@@ -388,9 +388,22 @@ stages:
       - powershell: eng/build.ps1 -configuration Release -prepareMachine -ci -restore -binaryLogName Restore.binlog 
         displayName: Restore
 
-      - powershell: eng/build.ps1 -configuration Release -prepareMachine -ci -build -pack -publish -sign -binaryLogName Build.binlog
+      - powershell: eng/build.ps1 -configuration Release -prepareMachine -ci -build -pack -publish -sign -binaryLogName Build.binlog /p:DotnetPublishUsingPipelines=true
```

Comment:
By enabling v3 publishing, symbol publishing is no longer performed during the build. This is necessary since the v2 publishing does not support the exclusions file which is necessary for the SQLite linux .so files. This means we need to perform symbol publishing in an equivalent way as v3 does.